src/widgets/ot_popup.eliomi

```diff
@@ -89,3 +89,110 @@ val confirm :
+     onmouseover : ('a Js.t, Dom_html.mouseEvent Js.t)
+                   Dom_html.event_listener Js.writeonly_prop;
+     onmouseup : ('a Js.t, Dom_html.mouseEvent Js.t)
+                 Dom_html.event_listener Js.writeonly_prop;
+     onscroll : ('a Js.t, Dom_html.event Js.t)
+                Dom_html.event_listener Js.writeonly_prop;
+     parentNode : Dom.node Js.t Js.opt Js.prop;
+     previousSibling : Dom.node Js.t Js.opt Js.prop;
+     removeChild : Dom.node Js.t -> Dom.node Js.t Js.meth;
+     replaceChild : Dom.node Js.t ->
+                    Dom.node Js.t -> Dom.node Js.t Js.meth;
+     tabIndex : < set : int -> unit; .. > Js.gen_prop; .. >
+   as 'a)
+  Js.t ->
+  < tabIndex : < set : int -> unit; .. > Js.gen_prop; .. > Js.t ->
+  < tabIndex : < set : int -> unit; .. > Js.gen_prop; .. > Js.t ->
+  (< onclick : ('b Js.t, Dom_html.mouseEvent Js.t)
+               Dom_html.event_listener Js.writeonly_prop;
+     ondblclick : ('b Js.t, Dom_html.mouseEvent Js.t)
+                  Dom_html.event_listener Js.writeonly_prop;
+     ondrag : ('b Js.t, Dom_html.dragEvent Js.t)
+              Dom_html.event_listener Js.writeonly_prop;
+     ondragend : ('b Js.t, Dom_html.dragEvent Js.t)
+                 Dom_html.event_listener Js.writeonly_prop;
+     ondragenter : ('b Js.t, Dom_html.dragEvent Js.t)
+                   Dom_html.event_listener Js.writeonly_prop;
+     ondragleave : ('b Js.t, Dom_html.dragEvent Js.t)
+                   Dom_html.event_listener Js.writeonly_prop;
+     ondragover : ('b Js.t, Dom_html.dragEvent Js.t)
+                  Dom_html.event_listener Js.writeonly_prop;
+     ondragstart : ('b Js.t, Dom_html.dragEvent Js.t)
+                   Dom_html.event_listener Js.writeonly_prop;
+     ondrop : ('b Js.t, Dom_html.dragEvent Js.t)
+              Dom_html.event_listener Js.writeonly_prop;
+     onkeydown : ('b Js.t, Dom_html.keyboardEvent Js.t)
+                 Dom_html.event_listener Js.writeonly_prop;
+     onkeypress : ('b Js.t, Dom_html.keyboardEvent Js.t)
+                  Dom_html.event_listener Js.writeonly_prop;
+     onkeyup : ('b Js.t, Dom_html.keyboardEvent Js.t)
+               Dom_html.event_listener Js.writeonly_prop;
+     onmousedown : ('b Js.t, Dom_html.mouseEvent Js.t)
+                   Dom_html.event_listener Js.writeonly_prop;
+     onmousemove : ('b Js.t, Dom_html.mouseEvent Js.t)
+                   Dom_html.event_listener Js.writeonly_prop;
+     onmouseout : ('b Js.t, Dom_html.mouseEvent Js.t)
+                  Dom_html.event_listener Js.writeonly_prop;
+     onmouseover : ('b Js.t, Dom_html.mouseEvent Js.t)
+                   Dom_html.event_listener Js.writeonly_prop;
+     onmouseup : ('b Js.t, Dom_html.mouseEvent Js.t)
+                 Dom_html.event_listener Js.writeonly_prop;
+     onscroll : ('b Js.t, Dom_html.event Js.t)
+                Dom_html.event_listener Js.writeonly_prop;
+     tabIndex : < set : int -> unit; .. > Js.gen_prop; .. >
+   as 'b)
+  Js.t -> unit
```

Comment:
Can you factor out the method collections that have a standard name? For example, you can define appropriate class types that `inherit Dom.node`, `inherit Dom_html.eventTarget`, etc. A seemingly arbitrary collection of methods makes the interface harder to understand.